### DISCUSSION TITLE
Arena gear trading for items from raids.

### DISCUSSION THREAD

**Discussion by Bayraktar93** [chromiecraft/chromiecraft#6134] [Suggestions]
I suggest adding NPS which can trade arena gear for specific items from raids. The list of arena items will be updated as the content is progressed. This implementation will increase amount of raiding, reduce loot DC, also will increase quality of PvP.

**Comment by balleny** [comment]
and reduce amount of pvp played.
blizz introduced this feature in patch 2.4 as a catch up way.
